enterprise/internal/db/perms_store.go

```diff
@@ -1531,8 +1531,14 @@ func (s *PermsStore) Metrics(ctx context.Context, staleDur time.Duration) (*Perm
 
 	stale := s.clock().Add(-1 * staleDur)
 	q := sqlf.Sprintf(`
-SELECT COUNT(*) FROM user_permissions
-WHERE updated_at <= %s
+SELECT COUNT(*) FROM user_permissions AS perms
+WHERE
+	perms.user_id IN
+		(
+			SELECT users.id FROM users
+			WHERE users.deleted_at IS NULL
+		)
+AND	perms.updated_at <= %s
```

Comment:
having a tab in between here feels weird
```suggestion
AND perms.updated_at <= %s
```